package.json

```diff
@@ -8,13 +8,19 @@
         "LICENSE",
         "README.md"
     ],
+    "types": "types",
```

Comment:
I think you also need to add `types/index.d.ts` to the list of files that should be published on npm.

Reply:
It's not really needed (as long as the file is called `index.d.ts`) but it doesn't harm. So I added the full path here 😉